crimsobot/cogs/reactions.py

```diff
@@ -0,0 +1,183 @@
+        try:
+            fact_object = await FunFacts.get_by_subject(subject.lower().strip(), ctx.guild.id)
+        except NoFactsExist:
+            embed = c.crimbed(
+                title='OMAN',
+                descr=f'There are no facts about {subject}!',
+                footer='You can add one by using ">fact about [fact subject]; [some fun fact]"',
+                color_name='yellow',
+                thumb_name='weary',
+            )
+            await ctx.send(embed=embed, delete_after=18)
+            return
+
+        return_string = ''.join([
+            f'`Fact ID: {fact_object.uuid}` · ',
+            f"**Here's a fact about {fact_object.subject.upper()}**:\n\n",
+            f'{fact_object.funfact}\n\n',
+        ])
+
+        await ctx.send(return_string)
+
+    @fact.command(name='add')
+    async def fact_add(self, ctx: commands.Context, *, something: CleanMentions) -> None:
+        """Add a fact"""
+
+        # if there are any errors in user input, use this embed
+        error_embed = c.crimbed(
+            title='BAD AT COMPUTER',
+            descr="You didn't do that correctly! Try this:\n`>fact add your subject; your fact`",
+            footer='You can always check ">help fact" for more info!',
+            color_name='yellow',
+            thumb_name='weary',
+        )
+
+        # see if there are uploaded files or media
+        try:
+            link_list = []
+            for link in ctx.message.attachments:
+                link_list.append(link.url)
+            links = '\n'.join(link_list)
+        except IndexError:
+            links = ''
+
+        user_input = something.split(';', 1)
+
+        fact_subject = user_input.pop(0).strip().lower()
+        try:
+            fact = f'{user_input[0].strip()} {links}'
+        except IndexError:
+            await ctx.send(embed=error_embed, delete_after=18)
+            return
+
+        if len(fact_subject) < 1 or len(fact) < 1:
+            await ctx.send(embed=error_embed, delete_after=18)
+            return
+
+        new_fact = await FunFacts.create_fact(ctx.author, ctx.guild.id, fact_subject, fact)
+
+        if new_fact:
+            embed = c.crimbed(
+                title=None,
+                descr='Fact added!',
+                footer=f'Fact ID: {new_fact.uuid}'
+            )
+            await ctx.send(embed=embed)
+
+    @fact.command(name='inspect')
+    async def fact_inspect(self, ctx: commands.Context, fact_id: int) -> None:
+        # check if owner for server-specific override
+        owner = ctx.author.id in ADMIN_USER_IDS
+
+        try:
+            fact_object = await FunFacts.get_by_id(fact_id, ctx.guild.id, owner)
+        except DoesNotExist:
+            embed = c.crimbed(title=None, descr=f'Fact {fact_id} does not exist (at least not in this server)!')
+            await ctx.send(embed=embed, delete_after=18)
+            return
+
+        fact_adder = self.bot.get_user(fact_object.created_by.discord_user_id)
+
```

Comment:
I don't think the line above and below this one need to be separated - this empty line could be removed.